### PR TITLE
Unified Sharing RFC

### PR DESCRIPTION
**Pull request by nblumhardt**
[Rendered](https://github.com/nblumhardt/seq-tickets/blob/unified-sharing/rfcs/0718-unified-sharing.md)

### CODE REVIEW

File: rfcs/0xxx-unified-sharing.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+The following guiding principles have been established:
+
+ 1. Sharing is an explicit choice. It should be easy and comfortable to work in Seq without worrying about cluttering up shared resources. Creating a new shared resource, or modifying an existing one, should be an obvious and deliberate action.
+ 1. Collaboration is the default. While administrative users may need to be able to restrict editing of sensitive signals and dashboards, this is the exception rather than the rule. Users should be free to collaborate and improve these things by default.
+ 1. [Labels]() and [Signal Groups]() are the primary means of dealing with clutter. It's unwieldy to rely on "hiding" items to keep lists clean, as personal roles/focus may change regularly.
+
+Applying these, the model proposed by this RFC is:
+
+ * All items can be marked with an "owner", in which case they will be regarded as "personal"
+ * All newly-create items will initially be owned by their creator (except in API-driven/automation scenarios)
```
Comment (KodrAus) [inline rfcs/0xxx-unified-sharing.md]: nit: _newly-created_

File: rfcs/0xxx-unified-sharing.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+ * Any user can make a "personal" item into a "shared" item
+ * Administrative users can mark shared items as "restricted", in which case they can only be modified by administrative users
+ * Any user can edit, remove, or make personal any unrestricted "shared" item
```
Comment (KodrAus) [inline rfcs/0xxx-unified-sharing.md]: What does _make personal_ here mean?

File: rfcs/0xxx-unified-sharing.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ * Users will by default see all of their personal items, and all shared items, in each category
+ * Shared items can be "hidden", which corresponds with the current "Remove" function on signals
```
Comment (KodrAus) [inline rfcs/0xxx-unified-sharing.md]: Can shared items also be deleted?

Comment (nblumhardt) [reply rfcs/0xxx-unified-sharing.md]: I've updated the wording of the item two above this, to make clear that all items can be deleted (previously this incorrectly used the term "remove")

**Comment by nblumhardt** [comment]
Thanks for the feedback, @KodrAus  - updated 👍

**Pull request merged**